libcontainer/console_linux.go

```diff
@@ -13,7 +12,7 @@ import (
 // NewConsole returns an initalized console that can be used within a container by copying bytes
 // from the master side to the slave that is attached as the tty for the container's init process.
 func NewConsole(uid, gid int) (Console, error) {
-	master, err := os.OpenFile("/dev/ptmx", syscall.O_RDWR|syscall.O_NOCTTY|syscall.O_CLOEXEC, 0)
+	master, err := os.OpenFile("/dev/pts/ptmx", syscall.O_RDWR|syscall.O_NOCTTY|syscall.O_CLOEXEC, 0)
```

Comment:
`pts(4)`, `pty(7)`, `posix_openpt(3)`, and the [runtime-spec][1] etc. all talk about `/dev/ptmx` for the UNIX 98 master clone device.  I think you should leave this path alone here, and (wherever runC sets up those default devices) [use a symlink][2] to the [required devpts mount][3].  If runC doesn't handle the default devices/mounts yet, you should still be able to leave this path alone and rely on however the path was working until now ;).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config-linux.md#devices
[2]: https://www.kernel.org/doc/Documentation/filesystems/devpts.txt
[3]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config-linux.md#default-file-systems